question-pages/batman-question/question1.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Batman-Question-1</title>
+    <script
+      src="https://kit.fontawesome.com/cbde1196cc.js"
+      defer
+      crossorigin="anonymous"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cosmic-ui.netlify.app/Components/cosmicUI.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cosmic-ui.netlify.app/Docs-css/Docs-navbar.css"
+    />
+    <link rel="stylesheet" href="https://cosmic-ui.netlify.app/style.css" />
+    <link rel="stylesheet" href="../../styles/homepage.css" />
+    <link rel="stylesheet" href="../../styles/rules.css" />
+    <link rel="stylesheet" href="../../styles/questionpage.css" />
+  </head>
+  <body>
+    <nav class="nav-bar boxShadow quizme-custom-nav">
+      <div class="nav-innerContainer quizme-inner-container">
+        <h2 class="nav-heading">QuizMe</h2>
+      </div>
+      <div class="nav-innerContainer nav-icon-container">
+        <a
+          class="quizme-custom-nav-clr"
+          href="https://www.linkedin.com/in/abhishek-sahani-698b12135/"
+          ><i class="fa-brands fa-linkedin icon-resize"></i
+        ></a>
+        <a
+          class="quizme-custom-nav-clr nav-icon"
+          href="https://github.com/abhisheksahani-nova"
+          ><i class="fa fa-github nav-icon"></i
+        ></a>
+      </div>
+    </nav>
+
+    <section class="d-flex justify-cont-center">
+      <div class="quizme-questionpages-container">
+        <div class="d-flex justify-cont-spacebetween mb-2">
+          <small>Questions: 1/5</small>
+          <small>Score: 0</small>
+        </div>
+
+        <p class="f-size-larger mb-2">
+          1. Who was introduced as Batman first ever foe in Batman #1 in April,
+          1940 ?
```

Comment:
I think rather than creating separate pages for questions, It is ideal to create a object to store questions and answers.